src/components/UserRegistration.js

```diff
@@ -0,0 +1,450 @@
+                            inline
+                            onChange={handleChange}
+                            />
+                            <Form.Check
+                            name="gender"
+                            id="formGender2"
+                            value="F"
+                            type="radio"
+                            label="F"
+                            inline
+                            onChange={handleChange}
+                            />
+                            <Form.Control.Feedback>Looks good!</Form.Control.Feedback>
+                            <Form.Control.Feedback type="invalid">
+                                Please provide a valid option 
+                            </Form.Control.Feedback>
+                        </Form.Group> 
+
+                        <Form.Group className="mb-3" controlId="formDob">
+                            <Form.Label><b>Date of birth</b></Form.Label>
+                            <Form.Control 
+                                required
+                                type="text" 
+                                name="dob" 
+                                value={values.dob}
+                                pattern="\d{1,2}/\d{1,2}/\d{4}"
+                                placeholder="Enter date of birth" 
+                                onChange={handleChange}
+                            />
+                            <Form.Text className="text-muted">
+                                ex. MM/DD/YYYY
+                            </Form.Text>
+                            <Form.Control.Feedback>Looks good!</Form.Control.Feedback>
+                            <Form.Control.Feedback type="invalid">
+                                Please provide a valid date of birth
+                            </Form.Control.Feedback>
+                        </Form.Group> 
+
+                        <Form.Group className="mb-3" controlId="formSsn">
+                            <Form.Label><b>Social security number</b></Form.Label>
+                            <Form.Control 
+                                required
+                                type="text" 
+                                name="ssn" 
+                                value={values.ssn}
+                                pattern="^\d{3}-?\d{2}-?\d{4}$"
+                                placeholder="Enter social security number" 
+                                onChange={handleChange}
+                            />
+                            <Form.Text className="text-muted">
+                                ex. 123-45-6789
+                            </Form.Text>
+                            <Form.Control.Feedback>Looks good!</Form.Control.Feedback>
+                            <Form.Control.Feedback type="invalid">
+                                Please provide a valid social security number
+                            </Form.Control.Feedback>
+                        </Form.Group>
```

Comment:
You can probably create a generic `FormInputField` to handle first name, last name, date of birth, and SSN.

Reply:
The same comment applies to most of the fields that come after line 199.